sienna_database_gen/pydantic_schemas.py

```diff
@@ -165,19 +165,29 @@ class Area(ObjModel):
     _table_name: ClassVar[Table] = "area"
 
 
-class Arc(ObjModel):
+class Arc(BaseModel):
+    id: Annotated[int, Column(Integer, primary_key=True)]
     from_id: Annotated[
         int, Column(Integer, ForeignKey("balancing_topology.id"), nullable=False)
     ]
     to_id: Annotated[
         int, Column(Integer, ForeignKey("balancing_topology.id"), nullable=False)
     ]
-    _table_name: ClassVar[Table] = "transmission_line"
+    _table_name: ClassVar[Table] = "arc"
```

Comment:
Do we need this or can we use `model_parametrized_name`?

Reply:
Doesn't appear work. I'm also partial to writing the names ourselves, even if there are sometimes mistakes.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Caskroom/miniconda/base/envs/ntp_griddb/lib/python3.12/site-packages/pydantic/main.py", line 498, in model_parametrized_name
    raise TypeError('Concrete names should only be generated for generic models.')
TypeError: Concrete names should only be generated for generic models.
```